wallet/src/de/schildbach/wallet/ui/BuyAndSellViewModel.kt

```diff
@@ -193,7 +193,7 @@ class BuyAndSellViewModel
                     _coinbaseBalance.value = response.value?.balance?.amount
                 }
                 is ResponseResource.Failure -> {
-                    _coinbaseBalance.value =  if (config.lastCoinbaseBalance.isNullOrEmpty()) {
+                    _coinbaseBalance.value = if (!config.lastCoinbaseBalance.isNullOrEmpty()) {
```

Comment:
This is an error. If `lastCoinbaseBalance` is not empty, we want to return it and not 0.